server/src/main/proto/spine_examples/kanban/commands.proto

```diff
@@ -36,15 +36,37 @@ option java_multiple_files = true;
 
 import "spine_examples/kanban/identifiers.proto";
 import "spine_examples/kanban/wip_limit.proto";
+import "spine_examples/kanban/column_position.proto";
 
 // A command to create a new board.
 message CreateBoard {
 
     // The ID of the board to create.
+    //
+    // Example note: By convention the first field of a command is its target.
```

Comment:
I don't understand why we need this note. Please remove. 

We either write a book, which implies leaving "notes" like this one everywhere, or we demonstrate how things are done. The former would be madness. The latter is the way to go.